httpcore5/src/main/java/org/apache/hc/core5/http/impl/bootstrap/ServerBootstrap.java

```diff
@@ -346,6 +347,16 @@ public final ServerBootstrap addFilterLast(final String name, final HttpFilterHa
         return this;
     }
 
+    /**
+     * Create {@link RequestRouter} with LOCAL_AUTHORITY_RESOLVER (default: IGNORE_PORT_AUTHORITY_RESOLVER).
```

Comment:
@cdw8j  Please do not mention `LOCAL_AUTHORITY_RESOLVER` and  `IGNORE_PORT_AUTHORITY_RESOLVER`. This is an implementation details and it should be mentioned in javadocs